problem_builder/slider.py

```diff
@@ -172,3 +172,10 @@ def validate_field_data(self, validation, data):
         Validate this block's field data.
         """
         super(SliderBlock, self).validate_field_data(validation, data)
+
+    def student_view_data(self):
```

Comment:
@pomegranited 

> So my question is, why isn't SliderBlock an eligible block type? And MRQBlock too for that matter?

IIRC, we implemented these data exports for Harvard GSE, and at the time they only wanted support for specific question types. MRQ was not one of them, and sliders didn't exist yet :)